examples/terraform-aws-s3-example/main.tf

```
@@ -53,6 +53,16 @@ data "aws_iam_policy_document" "s3_bucket_policy" {
   }
 }
 
+resource "aws_s3_bucket" "test-bucket-logs" {
```

Our terraform style guide is to use snake case instead of kebob case for block labels.

```suggestion
resource "aws_s3_bucket" "test_bucket_logs" {
```